.github/workflows/pr_pre-commit.yaml

```diff
@@ -14,6 +17,17 @@ jobs:
       - uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
         with:
           python-version: '3.12'
+      - name: Setup Hadolint
+        shell: bash
+        run: |
+          set -xeuo pipefail
+
+          LOCATION="$HOME/hadolint"
```

Comment:
Does it need adding to the GITHUB_PATH?

Reply:
Yeah that's the thing I'm not sure about. I didn't find any information on that in their documentation.

Reply:
Indeed, `$HOME` didn't work, we need to use `$GITHUB_PATH`.